app/code/community/Bolt/Boltpay/Controller/Traits/WebHookTrait.php

```diff
@@ -103,17 +103,13 @@ protected function verifyBoltSignature($payload, $signature ) {
      */
     protected function sendResponse($httpCode, $data = array())
     {
-        ob_end_clean();
```

Comment:
We want to clean the output buffer here to get rid of extra output introduced by the merchant server that is not relevant to Bolt